install/docker.sh

```diff
@@ -1,15 +1,24 @@
-sudo apt install -y docker.io docker-buildx
+# Remove conflicting Docker packages
+sudo apt purge --auto-remove docker.io docker-buildx docker-doc docker-compose docker-compose-v2 podman-docker containerd runc
+
+# Add the official Docker repo
+sudo install -m 0755 -d /etc/apt/keyrings
+sudo wget -qO /etc/apt/keyrings/docker.asc https://download.docker.com/linux/ubuntu/gpg
+sudo chmod a+r /etc/apt/keyrings/docker.asc
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt update
+
+# Install Docker engine and standard plugins
+sudo apt install -y docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin docker-ce-rootless-extras
+
+# Give this user privileged Docker access
 sudo usermod -aG docker ${USER}
 
 # Limit log size to avoid running out of disk
-echo '{"log-driver":"json-file","log-opts":{"max-size":"10m","max-file":"5"}}' | sudo tee /etc/docker/daemon.json
+echo '{"log-driver":"json-file","log-opts":{"max-size":"10m","max-file":"5"}}' | sudo tee /etc/docker/daemon.json > /dev/null
```

Comment:
Is the `json-file` logging driver needed for a specific purpose? (i.e. are there tools that try to consume the files it produces directly?).

Generally, [no tools should access those files directly](https://docs.docker.com/config/containers/logging/json-file/?highlight=jsonfile), as they're designed to be internal (and exclusively accessed by ) the daemon, but I know there's some tools that do (so maybe there was a reason for using this);

<img width="781" alt="Screenshot 2024-06-11 at 12 08 26" src="https://github.com/basecamp/omakub/assets/1804568/b8d5c675-883f-4498-9be0-772fb37248ef">


If that's not the case, then the `local` logging driver may be a good alternative; it can be more performant, and does rotation by default; https://docs.docker.com/config/containers/logging/local/

Reply:
Opened: https://github.com/basecamp/omakub/pull/85